### ISSUE TITLE
if it is required, install the dotenv app-extension too

### ISSUE THREAD

**Issue by nothingismagick** [kpapro/app-extension-firebase-authentication#4]
See this page for an idea how to do that:

https://github.com/quasarframework/quasar-testing/blob/dev/packages/testing/install.js

**Comment by adamkpurdy** [comment]
OMG!!! Yes Yes.... I know you mentioned last Friday about this, but I couldn't seem to figure it out or find this. Thank you, and this will definitely be in there.

**Comment by adamkpurdy** [comment]
Unfortunately, this approach isn't going to be a viable solution as the harness variable at line 20 is just being passed to the over all test installer. The installer then a does conditional check and runs specific code for each test harness to install.
  
Basically I would have to rewrite / copy all of the dotenv code into a separate file into my ext. This isn't sustainable as the extension ecosystem grows and expands, and is a code smell in my opinion. I'm just going to have to leave the installation process of this dependency on the developer. It'd be nice to have dependant extensions be able to run in the process of installing app extensions, but I don't see a true example of this right now. Please correct me if I'm mistaken.

**Comment by nothingismagick** [comment]
No, that example was just to show you how to integrate `execa` with `quasar
ext --add @quasar/dotenv`

**Comment by adamkpurdy** [comment]
Yeah I got that working, but all that command would do was add the extension to the extensions.json file, not execute the installation. If you have specifics please add more context.

**Comment by nothingismagick** [comment]
If you didn't get any prompts, then I have to recommend you file an issue
upstream.
